test/helpers/storage.py

```diff
@@ -82,8 +82,10 @@ def check_constraint(self, constraint, required=True):
     def confirm_entering_cockpit_storage(self):
         self.browser.click("#cockpit-storage-integration-enter-storage-confirm")
 
-    def return_to_installation(self):
+    def return_to_installation(self, error=None):
```

Comment:
I am surprised I don't see the error set anywhere in the added tests.

Reply:
Omg I forgot. Repushing.